x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/manage_locations/empty_locations.tsx

```diff
@@ -33,47 +39,48 @@ export const EmptyLocations = ({
           onClick={() => {
             setIsAddingNew?.(true);
             dispatch(setManageFlyoutOpen(true));
+            dispatch(setAddingNewPrivateLocation(true));
           }}
         >
           {ADD_LOCATION}
         </EuiButton>
       }
       footer={
-        <>
-          <EuiTitle size="xxs">
-            <h3>{LEARN_MORE}</h3>
-          </EuiTitle>
+        <EuiText size="s">
+          {LEARN_MORE}{' '}
           <EuiLink href="#" target="_blank">
             {READ_DOCS}
           </EuiLink>
-        </>
+        </EuiText>
       }
     />
   );
 };
 
-const START_ADDING_LOCATIONS = i18n.translate(
-  'xpack.synthetics.monitorManagement.startAddingLocations',
-  {
-    defaultMessage: 'Start adding private locations',
-  }
-);
+const FIRST_MONITOR = i18n.translate('xpack.synthetics.monitorManagement.firstLocationMonitor', {
+  defaultMessage: 'In order to create a monitor, you will need to add a location first.',
+});
+
+const ADD_FIRST_LOCATION = i18n.translate('xpack.synthetics.monitorManagement.firstLocation', {
+  defaultMessage: 'Add your first private location',
+});
 
 const START_ADDING_LOCATIONS_DESCRIPTION = i18n.translate(
   'xpack.synthetics.monitorManagement.startAddingLocationsDescription',
   {
-    defaultMessage: 'Add your first private location to run monitors on premiss via Elastic agent.',
+    defaultMessage:
+      'Private locations allow you to run monitors from your own premises. They require an Elastic agent and Agent policy which you can control and maintain via Fleet.',
   }
 );
 
 const ADD_LOCATION = i18n.translate('xpack.synthetics.monitorManagement.addLocation', {
   defaultMessage: 'Add location',
 });
 
-const READ_DOCS = i18n.translate('xpack.synthetics.monitorManagement.readDocs', {
+export const READ_DOCS = i18n.translate('xpack.synthetics.monitorManagement.readDocs', {
   defaultMessage: 'Read the docs',
```

Comment:
```suggestion
  defaultMessage: 'read the docs',
```

<img width="359" alt="Screen Shot 2022-07-25 at 3 09 31 PM" src="https://user-images.githubusercontent.com/11356435/180855828-fadb5029-f3c1-4c05-892f-86278d67e115.png">